### PR TITLE
New imports.log

### PR DESCRIPTION
**Pull request by michaelkain**
- shanoir-imports.log introduced, slight modification on shanoir-events.log format
- one line of CSV style log, per import started: evaluate especially the percentage of DICOM Enhanced used during the imports
- shanoir-ng-import.log optimised: nifti conversion removed + warnings reduced
- better debug with remote application: change in all Dockerfile (for dev only)

### CODE REVIEW

**Review by jcomedouteau — COMMENTED** [review]
Works OK for me concerning the imports.log file.

There are still a lots of logs coming from:
2022-07-20 13:50:19.661  INFO 1 --- [http-nio-9903-exec-1] org.dcm4che3.media.DicomDirWriter        : M-UPDATE /tmp/3/9128558961024244515/DICOMDIR: add IMAGE Record

But I'm not sure we can remove them easily.
Jean-Côme

File: shanoir-ng-import/src/main/java/org/shanoir/ng/importer/model/ImportJob.java [hunk 1]
```diff
@@ -198,5 +198,38 @@ public Long getUserId() {
...
+			importType = "SHUP";
+		} else if (fromPacs) {
+			importType = "PACS";
+		} else {
+			importType = "UNSUPPORTED";
+		}
+		int numberOfSeries = 0;
+		String modality = "unknown";
+		boolean enhanced = false;
+		if (patients != null && !patients.isEmpty()) {
```
[the remark this answers is not in the document]
Comment (michaelkain) [reply shanoir-ng-import/src/main/java/org/shanoir/ng/importer/model/ImportJob.java:217]: Good proposal: much better. Will do it.

**Comment by michaelkain** [comment]
Will try that too.

**Review by jcomedouteau — APPROVED** [review]

**Comment by michaelkain** [comment]
Dear Jean-Come,
thank you for your good comments. Normally it should be fine now.
Cheers, Michael

**Pull request merged**